main/chains/index.js

```diff
@@ -95,7 +95,7 @@ class ChainConnection extends EventEmitter {
       if (allowEip1559 && 'baseFeePerGas' in block) {
         try {
           // only consider this an EIP-1559 block if fee market can be loaded
-          const feeHistory = await gasMonitor.getFeeHistory(10, [10])
+          const feeHistory = await gasMonitor.getFeeHistory(20, [10, 60])
```

Comment:
i increased these blocks when testing op chains but not sure we want to change this for all chains